demo/agent.cc

```diff
@@ -29,6 +31,11 @@ bool ParseCommandLine(int argc, char* argv[]) {
     if (arg.find(kArgUserSpecific) == 0) {
       path = kPathUser;
       user_specific = true;
+    } else if (arg.find(kArgDelaySpecific) == 0) {
+      delay = std::stoul(arg.substr(strlen(kArgDelaySpecific)));
+      if (delay > 30) {
```

Comment:
Maybe you want to specify somewhere that the maximum delay is 30 seconds?

Reply:
Update the --help doc to state a max of 30 seconds.